src/components/Blog.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Let's move all of these files to a folder called "Pages" under `src` and then create a folder for each of them respectively and put them there?